README.md

```diff
@@ -272,6 +272,22 @@ SpannerOptions options = SpannerOptions.newBuilder()
 This option can also be enabled by setting the environment variable
 `SPANNER_ENABLE_EXTENDED_TRACING=true`.
 
+#### OpenTelemetry API Tracing
```

Comment:
Do we consider this a beta feature or a stable feature? If we consider this a beta feature, I would add `@BetaApi` to `setEnableApiTracing` and mention it here.

Reply:
I don't see any specific reason for why we would not consider this a stable feature, other than the comments already mentioned regarding the possiblity of changing attribute keys. In many ways, this is just an extension of something that was already supported (`ApiTracer` with OpenCensus) using the newer tracing framework, that is also considered a stable feature.